sass/layout/_sidebar-left.scss

```
@@ -358,6 +358,70 @@
                 }
             }
         }
+
+        .dropdown-menu .SidebarCategorySortingMenu__ActionItem {
+            padding: 0 15px 0 12px;
+
+            .MenuItem__primary-text {
+                span.icon {
+                    width: 100%;
+                    padding-right: 5px;
+                }
+            }
+        }
+        .categorySortingMenu {
+            text-transform: capitalize;
+
+            .MenuItem {
+                width: 100%;
+            }
+
+            .categorySortingMenuDiv {
+                display: flex;
+                padding: 0 15px;
+                margin: 0;
+                justify-content: space-between;
+                align-items: center;
+            }
+
+            .sorting-menu-dropdown-icon { 
+                padding: 0;
+                margin: 0;
+                font-size: 18px;
+            }
+
+            .sorting-menu-icon {
+                padding-left: 0 ;
+                width: 0;
+            }
+            
+            .sorting-menu-checkbox {
+                padding-right: 30px;
+                width: 0;
+            }    
+            
+            .MenuGroup.menu-divider {
+                margin: 5px -32px 5px 0;
+            }
+
+            .SubMenuItemContainer:not(.hasDivider) {
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+            }
+
+            .sorting-menu-checkbox i {
+                color: var(--button-bg);
+            }
+
+            .selected {
+                text-transform: none;
+                font-size: 12px;
+                color: var(--center-channel-color-56);
```

Can we move towards using the RGB variables?
```suggestion
                color: rgba(var(--center-channel-color-rgb), 0.56);
```